src/anomalib/models/image/efficient_ad/lightning_model.py

```diff
@@ -62,6 +64,7 @@ class EfficientAd(AnomalyModule):
 
     def __init__(
         self,
+        imagenet_dir: Path | str = "./datasets/imagenette",
```

Comment:
@alexriedel1, a bit minor, but would you prefer this variable to be named `imagenet_dir` or `imagenette_dir`?

Reply:
`imagenet_dir` is a bit more comprehensive for future generations i guess

Reply:
Hello @samet-akcay , @alexriedel1, how about `model_dir`? I would love to know your suggestions as well.

Reply:
I think `penalty_loss_dataset_dir` is better, as it gives more information, @samet-akcay what do you think?

Reply:
If we choose `penalty_loss_dataset_dir`, we might need to rename some other variables as well such as 
https://github.com/openvinotoolkit/anomalib/blob/2c87edf81471b8591e725443bfac83b83878416e/src/anomalib/models/image/efficient_ad/torch_model.py#L356

If @alexriedel1 prefers `imagenet_dir` than `imagenette_dir`, then let's stick to `imagenet_dir` for now. If another refactor needed, we could work on it later on.

Thanks